kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriberServiceImpl.java

```diff
@@ -59,13 +55,11 @@ public void update(Subscriber subscriber) {
         while (subscriptionIterator.hasNext()) {
             subscription = subscriptionIterator.next();
 
-            if ((subscription.getSubscriptionPack().getType() == SubscriptionPackType.PREGNANCY) &&
-                    (oldLMP != subscriber.getLastMenstrualPeriod()))  {
+            if (subscription.getSubscriptionPack().getType() == SubscriptionPackType.PREGNANCY)  {
```

Comment:
looks like there's 2 spaces between ) and { . I'm surprised that syle-cop didn't lose it #eet #ocd